frontend/package.json

```diff
@@ -26,8 +28,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "npx prettier --check ./src/**/*.{js,jsx,ts,tsx,scss} && npx eslint ./src/**/*.{js,jsx,ts,tsx}",
-    "format": "npx prettier --write ./src/**/*.{js,jsx,ts,tsx,scss} && npx eslint --fix ./src/**/*.{js,jsx,ts,tsx}"
+    "lint": "npx prettier --check './src/**/*.{js,jsx,ts,tsx,scss}' && npx eslint './src/**/*.{js,jsx,ts,tsx}'",
+    "format": "npx prettier --write './src/**/*.{js,jsx,ts,tsx,scss}' && npx eslint --fix './src/**/*.{js,jsx,ts,tsx}'"
   },
```

Comment:
what does this change do?

Reply:
My shell was trying to expand the paths before passing them to npx resulting in an error.
Adding quotation marks means the shell treats it as a string and the command runs correctly.